src/Std/Time/Format.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Format.Basic
+import Std.Time.Notation
```

Comment:
I think it would be good of there were general `LocalDateTime.parse`, `ZonedDateTime.parse`, `LocalDate.parse` etc. functions that can handle any reasonable format throws at them. Is there something like this or is something like this planned?